pkg/master/master.go

```diff
@@ -897,6 +898,10 @@ func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
 	if c.APIResourceConfigSource.ResourceEnabled(version.WithResource("networkpolicies")) {
 		storage["networkpolicies"] = networkPolicyStorage
 	}
+	storageClassStorage := storageclassetcd.NewREST(restOptions("storageclasses"))
```

Comment:
Are we enabling this by default (implies Beta) or are we calling it Alpha and expecting --runtime-config to enable it.  And then MAYBE move to beta before cycle closes?